Lib/test/test_call.py

```diff
@@ -665,5 +666,52 @@ def __call__(self, *args):
                 self.assertEqual(expected, wrapped(*args, **kwargs))
 
 
+class A:
+    def method(self, x, y):
+        return x + y
+
+    @staticmethod
+    def static_method():
```

Comment:
I think it would be good to give `method()` and `static_method()` more descriptive names, like you do for `positional_only()` (e.g. `no_arguments()`, etc or whatever distinguishes it).